docs/DeveloperGuide.md

```diff
@@ -79,7 +79,7 @@ command `delete 3`.
 
 <img src="images\SDforDeletePerson.png" width="800">
```

Comment:
Since this is the PR you made 7 days ago, should be for the last tutorial... In that case you did a good job :)